libs/domain/picking/src/components/user-profile/user-profile.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from 'lit';
+
+export const userProfileComponentStyles = css`
+  :host {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+
+  .info-block {
+    flex-grow: 1;
+
+    .info-label {
```

Comment:
Why we have this nesting in css? And how it's working? :)

Reply:
On this case we are using only native elements:
```
 <div class="info-block">
        <dl>
          <dt class="info-label">${i18n('user.profile.employee-id')}</dt>
          <dd class="info-value">admin@spryker.com</dd>
        </dl>
      </div>
```

Reply:
Ok, let me take a look from a different point of view:
- Why do we need a nesting here? I mean exact `.info-block .info-label` selector.
- Is it working in FF?

Reply:
We need the nesting because `.info-block` could have in the future an iteraction of more than one `.info-label`.
Currently is working on fulfillment app.